plugins/3MFReader/WorkspaceDialog.py

```diff
@@ -222,8 +222,14 @@ def intentName(self) -> str:
         return self._intent_name
 
     def setIntentName(self, intent_name: str) -> None:
+        intent_name = intent_name.title()
         if self._intent_name != intent_name:
-            self._intent_name = intent_name
+            if intent_name == "":
+                self._intent_name = "Balanced"
+            elif intent_name == "Quick":
+                self._intent_name = "Draft"
+            else:
+                self._intent_name = intent_name
```

Comment:
doing things this way won't translate the strings, additionaly we have this file that contains (translated) intent names, maybe we can use that?

https://github.com/Ultimaker/Cura/blob/main/cura/Machines/Models/IntentTranslations.py

Reply:
b584efd021dec1b30488c21ac4a5a36311cf7feb